src/pagination/pagination.spec.ts

```diff
@@ -440,6 +440,74 @@ describe('ngb-pagination', () => {
       fixture.detectChanges();
       expectPages(fixture.nativeElement, ['« Previous', '1', '2', '3', '4', '+5', '6', '7', '» Next']);
     });
+
+    describe('invalid inputs', () => {
+
+      ['page', 'collectionSize', 'maxSize', 'pageSize'].forEach((input) => {
+
+        it(`should throw for invalid "${input}" value`, () => {
+          const fixture = createTestComponent(`<ngb-pagination [collectionSize]="collectionSize" 
+            [maxSize]="maxSize" [pageSize]="pageSize" [page]="page"></ngb-pagination>`);
+          fixture.componentInstance[input] = NaN;
+          expect(() => { fixture.detectChanges(); }).toThrow();
```

Comment:
Wouldn't it be good to verify the exact message here? At least for one of inputs.